sdk/eventhub/Microsoft.Azure.EventHubs/src/Primitives/AzureActiveDirectoryTokenProvider.cs

```diff
@@ -5,60 +5,23 @@ namespace Microsoft.Azure.EventHubs
 {
     using System;
     using System.Threading.Tasks;
-    using Microsoft.IdentityModel.Clients.ActiveDirectory;
-
+    
     /// <summary>
     /// Represents the Azure Active Directory token provider for the Event Hubs.
     /// </summary>
     public class AzureActiveDirectoryTokenProvider : TokenProvider
     {
-        readonly AuthenticationContext authContext;
-
-#if ALLOW_CERTIFICATE_IDENTITY
-        readonly ClientCredential clientCredential;
-        readonly ClientAssertionCertificate clientAssertionCertificate;
-#endif
         readonly string clientId;
-        readonly Uri redirectUri;
-        readonly IPlatformParameters platformParameters;
-        readonly UserIdentifier userIdentifier;
-
-        enum AuthType
-        {
-            ClientCredential,
-            UserPasswordCredential,
-            ClientAssertionCertificate,
-            InteractiveUserLogin
-        }
-
-        readonly AuthType authType;
-
-#if ALLOW_CERTIFICATE_IDENTITY
-        internal AzureActiveDirectoryTokenProvider(AuthenticationContext authContext, ClientCredential credential)
-        {
-            this.clientCredential = credential;
-            this.authContext = authContext;
-            this.authType = AuthType.ClientCredential;
-            this.clientId = clientCredential.ClientId;
-        }
+        readonly string audience;
```

Comment:
>audience [](start = 24, length = 8)

I think this audience field isn't needed. When .GetTokenAsync() is called, the audience is supplied, so no need to specify the audience when initializing the token provider

Reply:
Correct, removed.